coderd/experiments.go

```diff
@@ -0,0 +1,19 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/coderd/httpapi"
+)
+
+// @Summary Get experiments
+// @ID get-experiments
+// @Security CoderSessionToken
+// @Produce json
+// @Tags General
+// @Success 200 {array} string
```

Comment:
Should this say the type?

```suggestion
// @Success 200 {array} codersdk.Experiments
```

Reply:
Fixed!